datafusion/functions-array/src/rewrite.rs

```diff
@@ -61,91 +59,6 @@ impl FunctionRewrite for ArrayFunctionRewriter {
                 Transformed::yes(array_has_all(*right, *left))
             }
 
-            // Column cases:
```

Comment:
Should we do the same thing for the other operators (AtArrow)?

Reply:
yes!